LNURLVend.ino

```diff
@@ -238,6 +254,25 @@ void loop() {
 
 ///////////DISPLAY///////////////
 
+/**
+ * Adjust QR code screen brightness
+ */
+void adjustQrBrightness(String direction){
+  if(direction == "increase" && qrScreenBrightness >= 0) {
```

Comment:
Should this be <= 0

Reply:
Actually, what does this condition really do at all? The value is clamped below, so it's not really needed.